src/main/anovos/data_report/basic_report_generation.py

```diff
@@ -62,9 +62,11 @@ def anovos_basic_report(spark, idf, id_col='', label_col='', event_label='', out
     AA_funcs = [correlation_matrix, variable_clustering]
     AT_funcs = [IV_calculation, IG_calculation]
     all_funcs = SG_funcs + QC_rows_funcs + QC_cols_funcs + AA_funcs + AT_funcs
-
-    if local_or_emr == "local":
+    
+    if global_run_type == "local":
         local_path = output_path
+    elif global_run_type == "databricks":
+        local_path = "/dbfs/FileStore/tables/report_stats"
```

Comment:
@ranjanravish Is this `local_path` something a user might want to change?

Reply:
yeah, i removed this static path and added function for assigning this local path